### ISSUE TITLE
Migrate practice exercises to rescript

### ISSUE THREAD

**Issue by tejasbubane** [exercism/reasonml#183]
- [x] accumulate - #173 
- [x] acronym - #174 
- [x] all-your-base - #175 
- [x] allergies - #179 
- [x] anagram - #180 
- [x] armstrong-numbers - #182 
- [ ] binary-search
- [ ] bob
- [ ] change
- [x] hello-world - #172 
- [ ] isogram
- [ ] leap
- [ ] minesweeper
- [ ] pangram
- [ ] phone-number
- [ ] protein-translation
- [ ] raindrops
- [ ] resistor-color
- [ ] rna-transcription
- [ ] roman-numerals
- [ ] run-length-encoding
- [ ] space-age
- [ ] two-fer
- [ ] word-count

**Comment by JsonKim** [comment]
@tejasbubane Hi. I made a PR #219 to migrate to ReScript. Can you review? If you review it, I will try to work on other examples as well.

**Comment by kevgathuku** [comment]
Hello all,
Thanks for the effort in making these updates 🔥

How can we access them? For example, downloading the `anagram` exercise downloads the old version based on `reasonml`

```
exercism download --track=reasonml --exercise=anagram
```

and trying to specify it as `rescript` doesn't work too 
```sh
 exercism download --track=rescript --exercise=anagram
Error: The track you specified does not exist
```

Also now that we have options like [Melange](https://melange.re/) is this still effort worth it or would it be better to update the current exercises to use Melange which is fully compatible with ReasonML?

Happy to help out in any of these approaches

**Comment by BNAndras** [comment]
Did you already start the exercise before the anagram files were changed? You should have an upgrade the exercise banner if you go to the exercise page on the Exercism website. Click that and if you run the exercism download command again, you should see the new files.

**Comment by kevgathuku** [comment]
Hi @BNAndras thanks for the pointers.
Yes, I believe I have a recent version of the CLI installed
```sh
❯ exercism version
exercism version 3.5.4
```

After checking a bit further it seems the new rescript exercises live in the [`rescript`](https://github.com/exercism/reasonml/tree/rescript) branch so maybe that could be why they are not available from the CLI yet

**Comment by BNAndras** [comment]
Yeah, the CLI doesn't directly pull from Github. The main branch of this repo gets synced with the Exercism website, and the CLI downloads from the website. You could grab the files from the rescript branch though and solve them locally, but you won't be able to submit your solution to the website successfully.

All tracks are volunteer-built and managed so if you're interesting in helping out the migration, I'd recommend making a discussion thread on the Exercism forums (https://forum.exercism.org/c/programming/reasonml/110).

**Comment by kevgathuku** [comment]
Awesome. Thanks a lot once again for the detailed info.

I will definitely try it out locally and will also try to follow up on the discussion in the forums
